lighthouse-core/audits/viewport.js

```diff
@@ -38,8 +39,15 @@ class Viewport extends Audit {
    * @return {!AuditResult}
    */
   static audit(artifacts) {
-    const hasMobileViewport = typeof artifacts.Viewport === 'string' &&
-        artifacts.Viewport.includes('width=');
+    if (typeof artifacts.Viewport !== 'string') {
+      return Viewport.generateAuditResult({
+        rawValue: false
```

Comment:
This should be an error case:

```
debugString: 'Error in determining viewport',
rawValue: -1
```

see https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/audits/dobetterweb/no-datenow.js#L57-L59